### PR TITLE
Fix commands not being registered on servers

### PR DESCRIPTION
**Pull request by Runemoro**
https://github.com/FabricMC/fabric/issues/142

### CODE REVIEW

File: src/main/java/net/fabricmc/fabric/mixin/commands/MixinServerCommandManager.java [hunk 1]
```diff
@@ -37,8 +37,9 @@
 	@Inject(method = "<init>(Z)V", at = @At("RETURN"))
 	public void addMethods(boolean dedicated, CallbackInfo info) {
 		// TODO: Run before findAmbiguities
-		if(!dedicated){
```
Comment (asiekierka) [inline src/main/java/net/fabricmc/fabric/mixin/commands/MixinServerCommandManager.java]: Pretty sure the code is this way as we hook DedicatedServers elsewhere

Comment (Runemoro) [reply src/main/java/net/fabricmc/fabric/mixin/commands/MixinServerCommandManager.java]: They didn't work when I tried it, see #142.

Comment (modmuss50) [reply src/main/java/net/fabricmc/fabric/mixin/commands/MixinServerCommandManager.java]: This commit should have fixed it: https://github.com/FabricMC/fabric/commit/98c307909e319e174c34badd96630419abd57b59

**Comment by Boundarybreaker** [comment]
This should be fixed ASAP before 1.14 full release on Tuesday. Is it ready to go?

**Comment by modmuss50** [comment]
I dont think this PR fixes the issue, and if it does my fix that has broken since I made it needs reverting.

**Comment by asiekierka** [comment]
Cannot reproduce; command registration works for me. Pushed a test mod.

**Pull request closed without merging**